paraview/xmls/UserInterfaceBase.xml

```diff
@@ -0,0 +1 @@
+<ServerManagerConfiguration></ServerManagerConfiguration>
```

Comment:
I don't believe that file is even needed. 
(it's supposed to be used by standalone gui programs)